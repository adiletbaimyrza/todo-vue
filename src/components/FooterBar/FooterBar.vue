<template>
  <footer :class="$style['container']">drag and drop to reorder list</footer>
</template>

<script>
export default {
  name: 'FooterBar',
}
</script>

<style lang="scss" module>
.container {
  margin-top: 4em;
  color: #b1b1b1;

  &::first-letter {
    text-transform: capitalize;
  }
}
</style>
