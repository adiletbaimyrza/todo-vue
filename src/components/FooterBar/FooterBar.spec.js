import { shallowMount } from '@vue/test-utils'
import FooterBar from './FooterBar.vue'

/* global describe it expect */
describe('FooterBar.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(FooterBar)
    expect(wrapper.exists()).toBe(true)
    expect(wrapper.element).toMatchSnapshot()
  })

  it('renders the correct text', () => {
    const wrapper = shallowMount(FooterBar)
    expect(wrapper.text()).toBe('drag and drop to reorder list')
  })
})
