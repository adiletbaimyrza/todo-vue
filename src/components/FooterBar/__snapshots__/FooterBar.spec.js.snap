// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBar.vue renders correctly 1`] = `
<footer
  class="container"
  data-testid="FooterBar"
>
   drag and drop to reorder list 
</footer>
`;
